qiskit/transpiler/preset_passmanagers/builtin_plugins.py

```diff
@@ -496,6 +507,13 @@ class OptimizationPassManager(PassManagerStagePlugin):
 
     def pass_manager(self, pass_manager_config, optimization_level=None) -> PassManager:
         """Build pass manager for optimization stage."""
+
```

Comment:
IMO it would be cleaner to have a `DefaultOptimizationPassManager` too and just use `OptimizationPassManager` as a dispatch. But we can also leave this as is for now and maybe reconsider if we have multiple basis gate modes.